spacy/displacy/__init__.py

```
@@ -37,16 +46,18 @@ def render(docs, style='dep', page=False, minify=False, jupyter=IS_JUPYTER,
+    html = _html["parsed"]
     if jupyter:  # return HTML rendered by IPython display()
         from IPython.core.display import display, HTML
+
         return display(HTML(html))
     return html
 
 
-def serve(docs, style='dep', page=True, minify=False, options={}, manual=False,
-          port=5000):
+def serve(
+    docs, style="dep", page=True, minify=False, options={}, manual=False, port=5000
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)